packages/server/graphql/mutations/helpers/createTaskInService.ts

```diff
@@ -42,13 +43,15 @@ const createTaskInService = async (
     const jiraTaskRes = await createJiraTask(taglessContentJSON, cloudId, projectKey, atlassianAuth)
     if (jiraTaskRes.error) return {error: jiraTaskRes.error}
     const {issueKey} = jiraTaskRes
+    const integrationRepoId = IntegrationRepoId.join({cloudId, projectKey, service})
     return {
       integration: new TaskIntegrationJira({
         accessUserId,
         cloudId,
         issueKey
       }),
-      integrationHash: JiraIssueId.join(cloudId, issueKey)
+      integrationHash: JiraIssueId.join(cloudId, issueKey),
+      integrationRepoId
```

Comment:
When we create a new integration issue in a Sprint Poker meeting, we call `createTask` which calls `createTaskInService`. The `integration` and `integrationHash` provide info about the issue, but not some about the `repoIntegration`. I'm now returning the `repoIntegrationId` so that we can update the prev used repo integration cache and update the timestamp.